packages/@webex/internal-plugin-metrics/src/prelogin-metrics-batcher.ts

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable valid-jsdoc */
+
+import {uniqueId} from 'lodash';
+import Batcher from './batcher';
+import {prepareDiagnosticMetricItem} from './call-diagnostic/call-diagnostic-metrics.util';
+import {generateCommonErrorMetadata} from './utils';
+
+const PRE_LOGIN_METRICS_IDENTIFIER = 'Pre Login Metrics -->';
+
+const PreLoginMetricsBatcher = Batcher.extend({
+  namespace: 'Metrics',
```

Comment:
Is there a reason or purpose to assigning this `namespace` attribute?

This can be potentially dangerous (in the same way that assigning two elements the same ID attribute within an HTML document can be dangerous). [See Documentation for AmpState](https://github.com/AmpersandJS/ampersand-state).

Reply:
No reason, just following the pattern.